mcp/include/ec_motors.h

```diff
@@ -37,8 +37,8 @@
     _map.subindex = _subindex;\
 }
 // reaction wheel P.I.(D.) coefficients
-#define RW_DEFAULT_CURRENT_P    1500
-#define RW_DEFAULT_CURRENT_I    45
```

Comment:
are these actual defaults? Magic numbers? A comment on why they are set there would be useful especially since we have changed from one set to another

Reply:
They are gains for the current control loop, the amount of current applied by to the coils of the motor by the drive each second vs. the commanded drive current. They are separate from az/el control loop gains. They come from CME2. Comment added.